23_Arithmetic2UnitTest/main.cpp

```diff
@@ -207,28 +196,31 @@ class ArithmeticUnitTestApp final : public application_templates::BasicMultiQueu
 		for (auto subgroupSize=MinSubgroupSize; subgroupSize <= MaxSubgroupSize; subgroupSize *= 2u)
 		{
 			const uint8_t subgroupSizeLog2 = hlsl::findMSB(subgroupSize);
-			for (uint32_t workgroupSize = subgroupSize; workgroupSize <= MaxWorkgroupSize; workgroupSize += subgroupSize)
+			for (uint32_t workgroupSize = subgroupSize; workgroupSize <= MaxWorkgroupSize; workgroupSize *= 2u)
```

Comment:
also if the device has native subgroup,  you want to test both emulation and native https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/190#discussion_r2036155341

the whether to use native or not needs to come from a define